src/components/statistics/badges/BadgeSubojectivesCategoryCompletion.tsx

```diff
@@ -59,13 +55,13 @@ export const BadgeSubojectivesCategoryCompletion = ({
     return;
   };
 
-  const setFilterBadgesLevel = () => {
+  /*   const setFilterBadgesLevel = () => {
     if (skillsNumber >= 10 && skillsNumber < 20)
       setBadgeFilterCss(`${styles.filterSilver}`);
     if (skillsNumber >= 20 && skillsNumber < 30)
       setBadgeFilterCss(`${styles.filterGold}`);
     if (skillsNumber >= 30) setBadgeFilterCss(`${styles.filterDiamond}`);
-  };
+  }; */
```

Comment:
Why is this commented ?

Reply:
It wasn't used in my version I don't know why, so I commented to check the issue later. It will be add on the next push